suite-native/state/src/reducers.ts

```diff
@@ -0,0 +1,44 @@
+import { combineReducers } from '@reduxjs/toolkit';
+import { persistReducer } from 'redux-persist';
+
+import {
+    prepareAccountsReducer,
+    prepareBlockchainReducer,
+    prepareFiatRatesReducer,
+    prepareTransactionsReducer,
+} from '@suite-common/wallet-core';
+import { devicesReducer } from '@suite-native/module-devices';
+import { appSettingsReducer, appSettingsPersistWhitelist } from '@suite-native/module-settings';
+
+import { extraDependencies } from './extraDependencies';
+import { reduxStorage } from './storage';
+
+const transactionsReducer = prepareTransactionsReducer(extraDependencies);
+const accountsReducer = prepareAccountsReducer(extraDependencies);
+const fiatRatesReducer = prepareFiatRatesReducer(extraDependencies);
+const blockchainReducer = prepareBlockchainReducer(extraDependencies);
+
+export const appSettingsPersistConfig = {
```

Comment:
Please create some wrapper like this:

```tsx
export const preparePersistReducer = <TReducerInitialState extends {}>({
    reducer,
    persistedKeys,
    key,
}: {
    reducer: ReducerWithInitialState<TReducerInitialState>;
    persistedKeys: Array<keyof TReducerInitialState>;
    key: string;
}): any => {
    persistReducer({ key, storage: mmkvStorage, whitelist }, reducer);
};
```

This wrapper will ensure that we don't need pass storage everytime and also it will ensure type safety, because it will allow whitelist only keys that actually exists in reducers state.

Reply:
And perhaps not type the return as `any` :]

Reply:
And what about `typedPersistReducer` so it doesn't sound like it returns some sort of a callback?

Reply:
Yea if you remove `any` it should inherit type automatically which is fine.

It's not only typed but it's also injecting storage by default so I don't think that is good name. But I am not agains choosing different name.

Reply:
It's a nitpick but `prepare` sounds like it returns a callback for later use, however here it's essentially used **instead** of `persistReducer`.

Reply:
`typedPersistReducer` is good name. On the other hand, we started to use `prepareXY` suffix for a lot of functions in the app. Connected mainly with `'@suite-common/wallet-core'`. There are prepareXYReducer, prepareXYMiddleware... I know we still don't have any DEV guidelines but this seems like it started to come about organically. I usually don't have a strong preference for this "naming" stuff but hey, lets keep this with `prepare`. From my point of view, this is really a detail.

Reply:
Feel free to use `typedPersistReducer` it will be probably better in the end because as @dahaca mentioned it's maintain implicit relation to `persistReducer`